BuildTools/.swiftlint-source.yml

```diff
@@ -65,6 +65,17 @@ type_contents_order:
     - [subscript, other_method, ib_action]
     - type_method
 
+custom_rules:
+  overridden_only_calls_super:
+    included: ".*.swift"
+    regex: 'override func [^\n]*\{\n(\s*super\.[^\n]*\n(\s*\/\/[^\n]*\n)*|(\s*\/\/[^\n]*\n)+)\s*\}'
+    message: "Overridden methods which only call super can be removed"
+
+  final_class:
+    included: ".*.swift"
+    regex: "^class|^private class|^public class"
```

Comment:
My current regex simply checks if a line starts with `class`, `private class` or `public class`, and if so, that is a failure. So an `open class` doesn't fail, which is great.

I'm sure this regex can be improved, but I am not good enough with regex 😬